idea/static/idea/css/idea.css

```diff
@@ -365,18 +365,16 @@ ul.section-nav li.active a {
 }
 
 /* Sidebar */
-.sidebar {
-    //margin-top: 1em;
-}
+
 .sidebar ul {
-    list-style: none;
-    padding-left: 0;
+    list-style-type: square;
+    padding-left: 1.5em;
 }
 .ic-add-tags,
 .sidebar-nav {
     margin-bottom: 2.5em;
 }
-.sidebar .challenge-banner {
+.challenge-banner {
```

Comment:
I scoped this as `.idea-hero .challenge-banner` since I wasn't sure where else `.challenge-banner` appeared, but I'm not sure if that's necessary or not.

Reply:
Good catch. It's also used on the individual challenge details page. I'll harmonize those styles.

Reply:
Pushed an update. I promise it looks good on my local ;)